### PR TITLE
OME005: Add sample scripts

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Add scripts to create OME-TIFF files with sub-resolutions.

### CODE REVIEW

File: OME005/index.md [hunk 1]
```diff
@@ -333,3 +333,25 @@ required.  With the corresponding reader support, this would provide
...
+files with the best tile sizes and compression types.  These will be
+used to test the TIFF and OME-TIFF support for sub-resolutions in
+Bio-Formats and OME Files prior to the creation of a writer which can
+generate the files directly.
+
+Note that the scripts require a copy of Bio-Formats `showinf`
+on the `PATH`.   They also require a copy of libtiff on
+`LD_LIBRARY_PATH` and `tiffinfo` and `tiffset` on the `PATH`. libtiff
+must be a release > 4.0.9 for BigTIFF SUBIFDS support in `tiffset`;
+at the time of writing this means building a copy from git.
```
Comment (mtbc) [inline OME005/index.md]: Might want to set a reminder somewhere for coming back to this.

Comment (rleigh-codelibre) [reply OME005/index.md]: When 4.0.10 comes out, it will include the fix.  Isn't that already covered?

Comment (mtbc) [reply OME005/index.md]: I don't know: do we already have a reminder somewhere to come back and update this text after 4.0.10 is out?

**Comment by joshmoore** [comment]
I'd suggest we be careful about the level of maintenance that we commit to the design PRs. In fact, we might want to come back in the not too distance future and put explicit dates on all of them saying, "This is a snapshot at time X" 

cF

screen shot 2018-05-08 at 12 38 29
screen shot 2018-05-08 at 12 39 17

**Comment by mtbc** [comment]
Absolutely, in "might want to set a reminder" I certainly didn't mean to imply any actual *commitment*, it just seemed an easy idea for a small useful tweak.

**Pull request merged**